artifacts/contracts/shanghai-opcodes/ShanghaiOpcodes.sol/ShanghaiOpcodes.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
Since currently the solidity compilation is _not_ integrated there's no way to know that the Shanghai EVM was targetted _or_ that the `PUSH0` (among others) was generated.  Strongly suggest using the command line options to dump the targeted EVM version _and_ the assembly language into json compiler output and commit those files as well.  So then we can all understand that what claims to be tested is in fact tested.

Wouldn't hurt to put the actual solidity compiler command line in here somewhere too - perhaps as a comment in the sol file.